service/file_transaction_service.py

```diff
@@ -0,0 +1,103 @@
+import grpc
+import hwsc_file_transaction_svc_pb2
+import hwsc_file_transaction_svc_pb2_grpc
+from service import server, utility
+
+
+class FileTransactionService(hwsc_file_transaction_svc_pb2_grpc.FileTransactionServiceServicer):
+    """A FileTransactionService class contains services for handling file transactions."""
+
+    def __init__(self, server):
+        self.__server = server
```

Comment:
`self.__server` is the same `Server` from https://github.com/hwsc-org/hwsc-file-transaction-svc/pull/44/files#diff-5bc02cefb3ea9e27f1a6776eabd1935dR5